src/SDKs/SqlManagement/Sql.Tests/FailoverGroupCrudScenarioTests.cs

```diff
@@ -0,0 +1,182 @@
+                var fgInput = new FailoverGroup()
+                {
+                    ReadOnlyEndpoint = new FailoverGroupReadOnlyEndpoint()
+                    {
+                        FailoverPolicy = "Disabled",
+                    },
+                    ReadWriteEndpoint = new FailoverGroupReadWriteEndpoint()
+                    {
+                        FailoverPolicy = "Automatic",
+                        FailoverWithDataLossGracePeriodMinutes = 120,
+                    },
+                    PartnerServers = new List<PartnerInfo>()
+                    {
+                        new PartnerInfo() { Id = partnerServer.Id },
+                    },
+                    Databases = new List<string>(),
+                };
+                var failoverGroup = sqlClient.FailoverGroups.CreateOrUpdate(resourceGroup.Name, primaryServer.Name, failoverGroupName, fgInput);
+                SqlManagementTestUtilities.ValidateFailoverGroup(fgInput, failoverGroup, failoverGroupName);
+
+                // Create a database in the primary server
+                //
+                string databaseName = "testdb";
+                var dbInput = new Database()
+                {
+                    Location = primaryServer.Location
+                };
+                Database database = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, primaryServer.Name, databaseName, dbInput);
+
+                // Update failover group to add database and change Read-write failover policy.
+                //
+                var fgUpdateInput = new FailoverGroup()
+                {
+                    ReadOnlyEndpoint = new FailoverGroupReadOnlyEndpoint()
+                    {
+                        FailoverPolicy = "Disabled",
+                    },
+                    ReadWriteEndpoint = new FailoverGroupReadWriteEndpoint()
+                    {
+                        FailoverPolicy = "Manual",
+                    },
+                    PartnerServers = new List<PartnerInfo>()
+                    {
+                        new PartnerInfo() { Id = partnerServer.Id },
+                    },
+                    Databases = new List<string>()
+                    {
+                        database.Id,
+                    },
+                };
+                failoverGroup = sqlClient.FailoverGroups.CreateOrUpdate(resourceGroup.Name, primaryServer.Name, failoverGroupName, fgUpdateInput);
+                SqlManagementTestUtilities.ValidateFailoverGroup(fgUpdateInput, failoverGroup, failoverGroupName);
+
+                var sourceDatabase = sqlClient.Databases.Get(resourceGroup.Name, primaryServer.Name, databaseName);
+                var targetDatabase = sqlClient.Databases.Get(resourceGroup.Name, partnerServer.Name, databaseName);
+                Assert.NotNull(sourceDatabase.FailoverGroupId);
+                Assert.NotNull(targetDatabase.FailoverGroupId);
+
```

Comment:
Could you have a test where you call Failover? Seems like an important scenario for FailoverGroups ;)